src/VocabGamePlay.jsx

```diff
@@ -1,48 +1,62 @@
 import React, { Component } from 'react';
 import "./VocabGamePlay.scss";
-import {GameDataset} from "./VocabGameData.js";
+import { GameDataset } from "./VocabGameData.js";
+import WordCard from './WordCard';
 
 class VocabGamePlay extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            firstSelection: null,
-            secondSelection: null,
-            vocabDataSet: GameDataset,
-        }
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstSelection: null,
+      secondSelection: null,
+      vocabDataSet: GameDataset,
     }
-    handleClick(e, data) {
-      console.log(data);
-      data.isSelected = !data.isSelected;
-      if (this.state.firstSelection === null) {
+  }
+
+  handleClick(e, data) {
+    console.log(data);
+    data.isSelected = !data.isSelected;
+    if (this.state.firstSelection === null) {
+      this.setState({ firstSelection: data });
+    } else if (this.state.secondSelection === null) {
+      if (data === this.state.firstSelection) {
         this.setState({ firstSelection: data });
+        return;
       }
-      else if (this.state.secondSelection === null) {
-        if (data === this.state.firstSelection) {
-          this.setState({ firstSelection: data });
-          return;
-        }
-        this.setState({ secondSelection: data }, () => { this.checkMatch(this.state.firstSelection, this.state.secondSelection) });
-      }
-    }
-    checkMatch(first, second){
-       this.isCorrectUpdate(first.word===second.translation);
+      this.setState({ secondSelection: data }, () => { this.checkMatch(this.state.firstSelection, this.state.secondSelection) });
     }
+  }
+  
 
-    isCorrectUpdate(isMatch){
-        let first=this.state.firstSelection;
-        let second=this.state.secondSelection;
-        first.isCorrect=isMatch;
-        second.isCorrect=isMatch;
+  checkMatch(first, second) {
+    this.isCorrectUpdate(first.word === second.translation);
+  }
+
+  isCorrectUpdate(isMatch) {
+    let first = this.state.firstSelection;
+    let second = this.state.secondSelection;
+    first.isCorrect = isMatch;
+    second.isCorrect = isMatch;
 
-        setTimeout(()=> {this.setState({
-            firstSelection: null,
-            secondSelection: null,
-        })}, 2000);
+    setTimeout(() => {
+      this.setState({
+        firstSelection: null,
+        secondSelection: null,
+      })
+    }, 2000);
+  }
+
```

Comment:
Okay so if I'm understanding correctly, what we've got here so far doesn't handle what happens after we set isCorrect? Just checking to make sure I'm following along!